components.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>HTML components w/ css for future skills</title>
+    <link rel="stylesheet" href="css/layout-helpers.css">
+    <link rel="stylesheet" href="salesforce-subtheme/salesforce-subtheme.css">
+    <link rel="stylesheet" type="text/css" href="code-highlighting/monokai.css">
+  </head>
+  <body>
+
+    <!-- https://www.freeformatter.com/html-escape.html#ad-output is your friend -->
+
+    <h1>Component library</h1>
+    <h2>Future Skills base theme</h2>
+    <h3>Centred-image box</h3>
+    <pre><code class="html">
```

Comment:
Is this correct? Highlighting doesn't appear to work for me. Based on the docs should it be?
```suggestion
    <pre><code data-language="html">
```

Same applies to the second example.